### DISCUSSION TITLE
Manager does not appear after installation and restart, why can It be?

### DISCUSSION THREAD

**Discussion by klavacaplan** [Comfy-Org/ComfyUI-Manager#232] [Q&A]
I 've cloned the code, restared Comfy but Manager menu didn't appear. What can be wrong?

**Comment by ltdrdata** [comment]
1. Did ComfyUI-Manager appear as an "import fail" in the terminal logs?
2. If that's not the case, please check the log messages in the browser's dev console.

**Reply by klavacaplan** [reply]
No, no errors were there.
But for some reason anfter an hour or so ComfyUi, which was open in a browser all that time, lost connection. I refreshed the page and manager appeared by itself. Before this refresh or restart didn't help, don't know why it works now :)

**Reply by ltdrdata** [reply]
Do you use cloud environment?

**Reply by klavacaplan** [reply]
I now got another problem but it is also about updating and installing. This time I've tried to install custom nodes. I never see a progress of installation but only see a message about DB (attached a screenshot). I wait for more than an hour just in case and restart comfy because there is no other way to close the window. One node was instaled, but second time it didn't work and second node failed (another screenshot). How should it work in fact?
![Снимок экрана 2023-12-12 140650](https://github.com/ltdrdata/ComfyUI-Manager/assets/108933327/6e1b2957-b461-45c2-8156-fe6d695c2149)
![Снимок экрана 2023-12-12 140814](https://github.com/ltdrdata/ComfyUI-Manager/assets/108933327/caf9bb84-239c-4d51-b935-917f2d17dd1a)

**Reply by klavacaplan** [reply]
As I understand they right process is when node is installed in few seconds, and then I press buttion restart, but my Comfy just can't reconnect for some reason. I close the window saying that it's reconnecting and go to list of custom nodes to see what's there and then this window about DB shows up, which I can't close. And I restart Comfy and then node is installed or maybe not. 
![Снимок экрана 2023-12-12 141654](https://github.com/ltdrdata/ComfyUI-Manager/assets/108933327/abe454c1-da36-4ffe-b7f6-b50be5318d02)
![Снимок экрана 2023-12-12 140650](https://github.com/ltdrdata/ComfyUI-Manager/assets/108933327/4d955199-349e-4090-9bc9-b238e4997dff)

**Reply by ltdrdata** [reply]
That phenomenon indicates that ComfyUI has been terminated. Check the messages displayed in the terminal.

**Reply by klavacaplan** [reply]
Hi! I've checked and terminal says it can't fine a file, but I do see the the file in the folder.
![Снимок экрана 2023-12-14 092017](https://github.com/ltdrdata/ComfyUI-Manager/assets/108933327/bb60abb3-0882-41d8-93ca-8db641c7b8a5)
![Снимок экрана 2023-12-14 092058](https://github.com/ltdrdata/ComfyUI-Manager/assets/108933327/547d3f55-b9f2-4faa-b76c-2ceec2c03b74)

**Reply by ltdrdata** [reply]
I recommend that don't use whitespace for installation path.

**Reply by wxseng** [reply]
> 1. Did ComfyUI-Manager appear as an "import fail" in the terminal logs?
> 2. If that's not the case, please check the log messages in the browser's dev console.

how to deal with the first situation?

**Reply by ltdrdata** [reply]
> > 1. Did ComfyUI-Manager appear as an "import fail" in the terminal logs?
> > 2. If that's not the case, please check the log messages in the browser's dev console.
> 
> how to deal with the first situation?

If an import fail occurs, you need to look at the error cause in the terminal log to diagnose the issue.

**Comment by klavacaplan** [comment]
> Do you use cloud environment?

no, or maybe I don't know what is cloud enviroment
